File: nathan/Problema0243/problema0243.py
vA = []
vB = [] 
vC = []

print("VETOR A") 
for i in range(0, 20): 
    n = int(input("Digite o numero na posição %d:" %(i+1))) 
    vA.append(n) 

print("VETOR B") 
for i in range(0, 20): 
    n = int(input("Digite o numero na posição %d:" %(i+1))) 
    vB.append(n) 

for i in range(0,20): 
    n = int(vA[i] - vB[i]) 
    vC.append(n) 
print(vC)

File: nathan/Problema0243/problema0243.java
package Problema0243;

import java.util.Scanner;

public class problema0243{
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);

        int vA[] = new int[20];
        int vB[] = new int[20];
        int vC[] = new int[20];

        System.out.println("VETOR A");
        for(int i = 0; i < 20; i++){
            System.out.println("Digite o numero da posição " + (i+1) + ": ");
            vA[i] = Integer.valueOf(input.nextLine());
        }
        System.out.println("VETOR B");
        for(int i = 0; i < 20; i++){
            System.out.println("Digite o numero da posição " + (i+1) + ": ");
            vB[i] = Integer.valueOf(input.nextLine());
        }
        for(int i = 0; i < 20; i++){
            vC[i] = vA[i] - vB[i];
        }
        for(int i = 0; i < 20; i++){
            System.out.println(vC[i] + " ");
        }
    }
}